docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+Work with your vendor to ensure these inspection and observability strategies work within your site-to-site architecture.
+
+### How can I prevent ngrok from being used for any purpose other than in site-to-site connectivity with my vendor? {#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?}
+
+The best way to disallow other uses of ngrok on your network is working with your vendor to specify a [custom agent ingress address](/docs/agent/ingress/#customize-agent-ingress-address).
+
+Your vendor will configure their DNS to issue certificates for the custom address, such as `your-company.us.connect.your-vendor.com:443`. Then they’ll work with you to [reconfigure your ngrok agents](/docs/agent/ingress/) to utilize the custom ingress address.
+
+We can also provision dedicated IPs for your custom agent ingress address, allowing you to whitelist addresses unique to your site-to-site configuration. Please reach out to your vendor if you’re interested in dedicated IPs.
+
+At this point, you can block the default agent ingress address at `connect.ngrok-agent.com:443`, which all agents use by default to connect outbound to ngrok’s network. This address resolves to a dynamic set of IP addresses, and blocking it at your network prevents any usage outside of this site-to-site connectivity use case in partnership with your vendor, such as developers utilizing ngrok to tunnel local development environments to the public internet.
+
+You should also block the ngrok-managed public domains:
+
+- ngrok.app
+- ngrok.dev
+- ngrok-free.app
+- ngrok-free.dev
+- ngrok.io
+- ngrok.pizza
+- ngrok.pro
+
+### How can we mitigate the risk of abuse?
+
+ngrok has a multi-pronged strategy for combating malicious use of our network, including automated systems that flag suspicious activity. We also disincentivize abuse through product restrictions on free and unverified ngrok accounts.
+
+Work with your vendor to design an architecture that [prevents unauthorized use](#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?) and uses the appropriate [encryption model](#how-does-ngrok’s-traffic-encryption-work?) between your services.
+
+See our [abuse page](https://ngrok.com/abuse) for details.
+
+## Privacy and compliance
+
+### What compliance certifications does ngrok maintain?
+
+ngrok complies with SOC 2, GDPR, EU-US DPF, and CCPA.
+
+You can request our SOC 2 report and view other documents about ngrok’s security measures, like annual penetration testing, on our [Trust Center](https://trust.ngrok.com/).
+
+### What data does ngrok have access to, and how long is it stored?
+
+ngrok’s access to your data depends on which [encryption model](#how-does-ngrok’s-traffic-encryption-work?) specified by your site-to-site connectivity architecture—reach out to your vendor for more details.
```

Comment:
ngrok’s access to your data depends on the [encryption model](#how-does-ngrok’s-traffic-encryption-work?) specified by your site-to-site connectivity architecture—reach out to your vendor for more details.